server/routerlicious/packages/services/src/tenant.ts

```diff
@@ -245,7 +249,34 @@ export class TenantManager implements core.ITenantManager, core.ITenantConfigMan
 		return gitManager;
 	}
 
+	private convertInvalidTokenErrorToNetworkError(cachedInvalidTokenError: string): void {
```

Comment:
Lets rename this to `throwInvalidTokenErrorAsNetworkError`, or switch it to return the NetworkError, then below, do `throw this.converInvalidTokenErrorToNetworkError(...)`